service/docs/source/geopmpy.7.rst

```diff
@@ -20,9 +20,9 @@ In addition, there is 1 infrastructure module provided:
 io.py
 ^^^^^
 This module provides tools for parsing and encapsulating report and trace data
-into either simple structures or Pandas DataFrames.  It can be used to parse
+into either simple structures or **Pandas** ``DataFrame``\ s.  It can be used to parse
```

Comment:
Could alternatively link to `pandas.DataFrame` using a `:py:class:` reference. Pandas is part of our intersphinx list so it should automatically figure out the right URL.

Reply:
> Could alternatively link to `pandas.DataFrame` using a `:py:class:` reference. Pandas is part of our intersphinx list so it should automatically figure out the right URL.

This is in a `*.rst` file though, not in a python docstring.

Reply:
Here's the Sphinx documentation about how to use that feature in rst files: https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html?highlight=%3Apy%3Ameth#role-py-func